### PR TITLE
Pools support

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: tracker/src/Tracker/Models/Interop/Pool.hs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import RIO
+
+import ErgoDex.Amm.Orders
+import ErgoDex.Contracts.Types as Currencies
+import ErgoDex.State
+import ErgoDex.Amm.Pool
+import ErgoDex.Class
+import ErgoDex.Types
+
+data Pool = Pool
```
Comment (Bromel777) [inline tracker/src/Tracker/Models/Interop/Pool.hs:17]: why not ErgoDex.Amm.Pool?

Comment (GusevTimofey) [reply tracker/src/Tracker/Models/Interop/Pool.hs:17]: Cause I did an interop module for better json serialisation(`ErgoDex.Amm.Pool` contains models from plutus which json codecs are awful)

File: tracker/src/Tracker/Models/Events/ExecutedOrderEvent.hs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import Tracker.Models.Interop.Pool
+
+import Data.Aeson
+import GHC.Generics
+import Prelude 
+import RIO
+
+data ExecutedOrderEvent = ExecutedOrderEvent
+  { stringJson :: String
```
[the remark this answers is not in the document]
Comment (GusevTimofey) [reply tracker/src/Tracker/Models/Events/ExecutedOrderEvent.hs:13]: It consists of Executed order/swap/redeem serialised into json and then transformed into the string

File: streaming-kafka/src/Streaming/Class.hs [hunk 3]
```diff
@@ -1,11 +1,31 @@
+{-# LANGUAGE FlexibleInstances #-}
```
[the remark this answers is not in the document]
Comment (GusevTimofey) [reply streaming-kafka/src/Streaming/Class.hs]: moved

**Pull request merged**